contrib_bots/lib/myanimelist.py

```diff
@@ -0,0 +1,95 @@
+import requests, logging
+from xml.etree import ElementTree
+
+# Signup for a MAL account here https://myanimelist.net/register.php?from=%2F
+USERNAME = '[PLEASE CHANGE]'
```

Comment:
Not sure how would I go about passing a credentials file to the bot script since the parser is handled by run.py.